config.yml

```diff
@@ -235,10 +235,11 @@ HOME_FEATURES:
     type: VerticalCardList
 
 READ_TAB:
-  - algorithms:
+  - subtitle: Read
+    algorithms:
       - type: CONTENT_CHANNEL
         arguments:
-          contentChannelId: 18
+          contentChannelId: 17
           limit: 1
     type: VerticalCardList
   - title: Continue Reading
```

Comment:
```suggestion
  - subtitle: Continue Reading
```

Reply:
I can't comment on line 251 - but that should probably also be `title`

Reply:
the mock shows different sizes is why I did it.

Reply:
ah - in this case I think the all the same size looks better than using the `title`s for the two lower features.

Reply:
done